docs/chapters/workflow/release/release-process.rst

```diff
@@ -9,10 +9,36 @@ There are two types of releases, source releases, and binary releases. A binary
 release has all the requirements of a source release and more. This split is for
 licensing reasons, as there are more obligations when doing a binary release.
 
+Binary release
+--------------
+Binaries of pelux are released on each process and can be found under our `downloads page`_.
```

Comment:
Can be re-worded to
Binary images for Pelux 3.0 are now available for download (we give a hyperlkink for download). Each new release will have a set of binary images supporting the various variants as mentioned below

Reply:
If we mention "3.0" at all I think it should be something like "Binary images are available for download since the release of PELUX 3.0..."